rasa/cli/arguments/default_arguments.py

```diff
@@ -156,3 +156,10 @@ def add_logging_options(parser: argparse.ArgumentParser) -> None:
         dest="loglevel",
         const=logging.WARNING,
     )
+
+    logging_arguments.add_argument(
+        "--logging-config-file",
```

Comment:
how about providing a default location and `try`ing to load the file?

Reply:
Does that also involve adding an empty file in the `rasa init` project structure?
Or just mention in the docs that we expect the file to be named in a certain way?

Reply:
> Does that also involve adding an empty file in the rasa init project structure?

No I don't think so at this stage. 

> Or just mention in the docs that we expect the file to be named in a certain way?

Yes - if we do that, the doc should mention that the option looks for a filename by default, but people can provide the file they want

Reply:
I tried exploring this, but in the end abandoned this, as it required quite a lot of small changes in different places which I felt made the code too messy, especially that it's not a file that comes with `rasa init` yet.